packages/medusa/src/api-v2/admin/products/helpers.ts

```diff
@@ -21,7 +24,10 @@ export const remapKeysForVariant = (selectFields: string[]) => {
   const pricingFields = selectFields
     .filter((fieldName: string) => fieldName.startsWith("prices"))
     .map((fieldName: string) =>
-      fieldName.replace("prices.", "price_set.money_amounts.")
+      fieldName.replace(
+        "prices.",
+        "price_set.price_set_money_amounts.money_amount."
```

Comment:
Hey @sradevski, just as an FYI, I've severed the priceSet <> money amounts relationship, if you're working around this, you'll have to get to money amount through price_setmoney_amount, which will be moved to price_set.prices eventually.

Reply:
@riqwan I think I'll just wait for the final version and in the meantime I can switch to working on something else. Thanks for the heads up 👍